### PR TITLE
PT-13027 - fix code-style and phpstan issues

### PR DESCRIPTION
**Pull request by ennasus4sun**

### CODE REVIEW

**Review by mitelg — CHANGES_REQUESTED** [review]
plugin.xml entry was also missing for the contribution

File: plugin.xml [hunk 1]
```diff
@@ -5,13 +5,22 @@
...
     <copyright>(c) by shopware AG</copyright>
     <license>proprietary</license>
-    <link>http://store.shopware.com</link>
+    <link>https://store.shopware.com</link>
     <author>shopware AG</author>
     <compatibility minVersion="5.6.0" maxVersion="5.99.99"/>
 
+    <changelog version="3.0.3">
+        <changes lang="de">
+            PT-13027 - Hinzufügen der Variable sTaxRates zur sOrder-Mail;
```
Comment (mitelg) [inline plugin.xml:17]:
```suggestion
            PT-13027 - Die Steuern sind nun im Template der Bestellbestätigungsmail verfügbar;
```

please make sure, that this is not to technical, as this is for the shop owners/ merchants

File: plugin.xml [hunk 2]
```diff
@@ -5,13 +5,22 @@
...
+        </changes>
+        <changes lang="en">
+            PT-13027 - Add sTaxRates-variable to sOrder mail;
```
Comment (mitelg) [inline plugin.xml:20]:
```suggestion
            PT-13027 - Taxes are now available in the order confirmation email template;
```

File: plugin.xml [hunk 3]
```diff
@@ -5,13 +5,22 @@
     <label lang="de">Backend Bestellungen</label>
     <label lang="en">Backend orders</label>
 
-    <version>3.0.2</version>
+    <version>3.0.3</version>
```
Comment (mitelg) [inline plugin.xml:8]:
```suggestion
    <version>3.1.0</version>
```

changelog sounds like a feature to me :slightly_smiling_face:

File: plugin.xml [hunk 4]
```diff
@@ -5,13 +5,22 @@
...
     <copyright>(c) by shopware AG</copyright>
     <license>proprietary</license>
-    <link>http://store.shopware.com</link>
+    <link>https://store.shopware.com</link>
     <author>shopware AG</author>
     <compatibility minVersion="5.6.0" maxVersion="5.99.99"/>
 
+    <changelog version="3.0.3">
```
Comment (mitelg) [inline plugin.xml:15]:
```suggestion
    <changelog version="3.1.0">
```

**Review by mitelg — CHANGES_REQUESTED** [review]

**Review by swDennis — APPROVED** [review]

**Pull request merged**